localstack/services/opensearch/versions.py

```diff
@@ -214,16 +216,41 @@
     ),
     CompatibleVersionsMap(
         SourceVersion="OpenSearch_1.3",
-        TargetVersions=["OpenSearch_2.3", "OpenSearch_2.5", "OpenSearch_2.7", "OpenSearch_2.9"],
+        TargetVersions=[
+            "OpenSearch_2.3",
+            "OpenSearch_2.5",
+            "OpenSearch_2.7",
+            "OpenSearch_2.9",
+            "OpenSearch_2.10",
```

Comment:
AWS does not list 2.10 in their supported versions:
```
aws --region us-east-1 opensearch list-versions                                                                                                                                                                                                    11:26:20
{
    "Versions": [
        "OpenSearch_2.11",
        "OpenSearch_2.9",
        "OpenSearch_2.7",
        "OpenSearch_2.5",
        "OpenSearch_2.3",
        "OpenSearch_1.3",
        "OpenSearch_1.2",
        "OpenSearch_1.1",
        "OpenSearch_1.0",
        "Elasticsearch_7.10",
        "Elasticsearch_7.9",
        "Elasticsearch_7.8",
        "Elasticsearch_7.7",
        "Elasticsearch_7.4",
        "Elasticsearch_7.1",
        "Elasticsearch_6.8",
        "Elasticsearch_6.7",
        "Elasticsearch_6.5",
        "Elasticsearch_6.4",
        "Elasticsearch_6.3",
        "Elasticsearch_6.2",
        "Elasticsearch_6.0",
        "Elasticsearch_5.6",
        "Elasticsearch_5.5",
        "Elasticsearch_5.3",
        "Elasticsearch_5.1",
        "Elasticsearch_2.3",
        "Elasticsearch_1.5"
    ]
}
```
Please remove 2.10 here and in the rest of the mapping, and generally make sure to verify your changes against AWS.